core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -269,6 +269,22 @@ trait AttributeComponent {
       }
     }
 
+    def findUniqueStringsByNameQuery(attrName: AttributeName, queryString: Option[String]) = {
+
+      val basicFilter = filter(rec =>
+        rec.namespace === attrName.namespace &&
+          rec.name === attrName.name &&
+          rec.valueString.isDefined)
+
+      val res = (queryString match {
+        case Some(query) => basicFilter.filter(_.valueString.like(s"%${query}%"))
+        case None => basicFilter
+      }).groupBy(_.valueString).map(queryThing =>
+        (queryThing._1, queryThing._2.length))
+
+      res.sortBy(r => (r._2.desc, r._1))
```

Comment:
one last tweak? You can put the `.get` back in after the sortBy, e.g.:
```
res.sortBy(r => (r._2.desc, r._1)).map(x => (x._1.get, x._2))
```
and then you're returning a String, not an Option[String], and calling code is simplified. This means you'd have to revert the `Some()`s you added in the most recent commit.